docs/source/tutorial/resolvers.md

```diff
@@ -99,11 +99,12 @@ By providing your resolver map to Apollo Server like so, it knows how to call re
 
 ## Run test queries
 
-Let's run a test query on our server! Return to GraphQL Playground by running `npm start` and visiting `http://localhost:4000/` in your browser:
+Let's run a test query on our server! Start it up with `npm start` and open whichever tool you previously used to [explore your schema](./schema/#explore-your-schema):
 
-<img class="screenshot" src="../images/playground.png" alt="GraphQL Playground"></img>
+* The Explorer in [Apollo Studio](https://studio.apollographql.com/)
```

Comment:
You'll want to add a `/dev` here I think. It has special behavior:
- logged out, brand new user will be taken to the streamlined signup form
- logged in user with no dev graphs will be taken to the new graph modal
- logged in user with at least one dev graph will be taken to the last dev graph they were looking at